src/sentry/static/sentry/app/components/gridEditable/index.tsx

```diff
@@ -368,7 +368,7 @@ class GridEditable<
       <GridRow>
         <GridBodyCellStatus>
           <EmptyStateWarning>
-            <p>{t('No results found')}</p>
+            <p>{t('No results found for the given query')}</p>
```

Comment:
```suggestion
            <p>{t('No results found for your query')}</p>
```

Reply:
bleh i think i actually already expressed opposition to this but i don't want to argue about it anymore so it's fine for now